### ISSUE TITLE
Binance API returns info too late

### ISSUE THREAD

**Issue by kamiljaworski975** [CyberPunkMetalHead/gateio-crypto-trading-bot-binance-announcements-new-coins#90]
Hello guys, do you know how is it possible that someone has access to new announcement like 20-40sec before Binance list it on that address: https://www.binance.com/bapi/composite/v1/public/cms/article/catalog/list/query?catalogId=48&pageNo=1&pageSize=20;

I've got info in my Scrapper about new listing: 23 2021 01:59:56 GMT+0000, but when you take a look into tradingview.com it started to rise at: 23 2021 01:59:23 GMT+0000.

Basically everything went fine, I've opened trade and still made some profit, but the first jump was skipped...

Do you have other url API to get that info faster

Edit: Is it because Binance bought that coin in order to sell it on their platform and then they display announcement? 
![Screenshot 2021-11-23 at 02 25 27](https://user-images.githubusercontent.com/53656878/142962868-7928881a-7061-4440-b096-40baaff4d6fa.png)
?

**Comment by denzilsaldanha** [comment]
I myself have been trying to write a bot that scrapes binance and then places a buy order. 

I've found two sites where listings come out and they differ by abit. 
I'm starting to feel https://www.binance.com/en/support/announcement/c-0 might list annoucements first.

**Comment by kamiljaworski975** [comment]
About scraping website I've tried that before, but results was much worse than calling the Binance API. I've tried all of the regions and results in all cases were worse about 1-3mins. For example todays listing was displayed on website 2mins and 36secs later than data in API.

**Comment by denzilsaldanha** [comment]
Fair enough, yeah that would make sense. Server response should be faster than scraping an api. I wonder if the delay in your response is due to caching. Have you tried setting a no-cache header?

**Comment by kamiljaworski975** [comment]
I haven't, I'll implement it. Basically I'm getting response like 50-200ms after call, but yeah there could be problem that my server is caching headers and sending the same response for some period of time. Need to verify it.
